Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -167,8 +172,8 @@
     for iTE = 1:xQ.NumberEchoTimes
         for iTI = 1:nDecodedTI
             
-            indexPositive = find(xQ.TimeEncodedMatrix(iTI,:)==1);
-            indexNegative = find(xQ.TimeEncodedMatrix(iTI,:)==-1);
+            indexPositive = find(xQ.TimeEncodedMatrix(iTI+1,:)==1);
+            indexNegative = find(xQ.TimeEncodedMatrix(iTI+1,:)==-1);
```

Comment:
Why not keeping the matrix square until here, and then add an extra first row with positives only?
Or is that the whole point, that not everybody has this?

Reply:
While the first row/column is in most cases all 1, everybody still prints it in papers/code/etc. 
So more common, easier, and less confusing to have that line in here as well.